tools/accuracy_checker/accuracy_checker/preprocessor/README.md

```diff
@@ -119,5 +119,6 @@ Accuracy Checker supports following set of preprocessors:
 * `resample_audio` - converts audio to new sample rate
   * `sample_rate` - sets new sample rate
 * `clip_audio` - slices audio into several parts with equal duration
-  * `duration` - sets duration in seconds
-  * `max_clips` - sets the maximum number of clips (by default `1`)
+  * `duration`, `duration_in_samples` - sets duration in seconds or samples respectively (only one acceptable)
```

Comment:
it is not clear in which cases which option should be used.
Also i think it can be combined in simmilar way like it done in `subsample size` (option accepts integer values and string with % at the end for relative partition, here for seconds can be used with s postfix for example)

Reply:
Thanks for comment. I delete duplication, see changes in Readme for more information.